### PR TITLE
improve spinner for remote deploy

### LINKED ISSUE #3359: Remote deployment - spinner update
I want to know how much data I'm being synchronising if it takes long to synchronise.
I want to get an spinner when retrieving build images and while checking if the app is deployed

## Acceptance criteria

- [ ] After 10 s synchronising data, show a waning message with a call to action to use the dockerignore for deployments
- [ ] Show how much data has been synchronised with remote
- [ ] Add spinner when checking if the images has been already built
- [ ] Add spinner when checking if the app is deployed

### PR DESCRIPTION
**Pull request by AdrianPedriza**
# Proposed changes

Fixes #3359 
- Display current context loaded.
- When a deployment load a context greater than 50MB, an info log is displayed.

https://user-images.githubusercontent.com/22500940/216581840-977e1c9b-0b16-412e-8a4c-1d2a9e172c8f.mp4

cc @rlamana @pchico83

### CODE REVIEW

**Comment by pchico83** [comment]
@AdrianPedriza I would always show the size of what's synched, it gives great feedback on what's going on.
I would remove the warning for large context. Instead, if the size is larger than 50MB, for example, I would show an info message saying that you can configure the `.oktetodeployignore` file to optimize your context

**Review by pchico83 — APPROVED** [review]

**Comment by pchico83** [comment]
@AdrianPedriza do you know why the output seems to flow in chunks every few seconds instead of getting it in live streaming?

**Comment by AdrianPedriza** [comment]
> @AdrianPedriza do you know why the output seems to flow in chunks every few seconds instead of getting it in live streaming?

On our side we are not grouping the logs coming from the construction so I would say it is a matter of how the messages are coming from the remote. I will review this logic in case we can improve it on our end

**Review by ifbyol — APPROVED** [review]

File: pkg/cmd/build/logger.go [hunk 1]
```diff
@@ -111,10 +117,20 @@ func (t *trace) update(ss *client.SolveStatus) error {
...
 
-func (t trace) display() {
+func (t *trace) display() {
 	for _, v := range t.ongoing {
+		if t.isTransferringContext(v.name) {
+			if v.currentTransferedContext != 0 {
+				currentLoadedCtx := units.Bytes(v.currentTransferedContext)
+				if t.showCtxAdvice && currentLoadedCtx > largeContextThreshold {
+					t.showCtxAdvice = false
+					oktetoLog.Information("You can use '.oktetodeployignore' file to optimize the context used to deploy your development environment.")
```
[the remark this answers is not in the document]
Comment (AdrianPedriza) [reply pkg/cmd/build/logger.go:127]: that video was related to the previous approach. This video fits with the latest commit on the PR :) 

https://user-images.githubusercontent.com/22500940/217537376-6e40353b-5cf1-4aac-b0a4-13b5eb6107c6.mp4

[the remark this answers is not in the document]
Comment (AdrianPedriza) [reply pkg/cmd/build/logger.go:127]: description updated

**Pull request merged**